packages/a11y/src/add-container.js

```diff
@@ -1,15 +1,13 @@
 /**
  * Build the live regions markup.
  *
- * @param {string} ariaLive Optional. Value for the 'aria-live' attribute, default 'polite'.
+ * @param {string} [ariaLive] Value for the 'aria-live' attribute; default: 'polite'.
  *
  * @return {HTMLDivElement} The ARIA live region HTML element.
  */
-const addContainer = function( ariaLive ) {
-	ariaLive = ariaLive || 'polite';
```

Comment:
Technically this would previously fall back to `'polite'` for any [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) value given (`null`, `false`, `0`, etc), and now it will only fall back when explicitly `undefined` (omitted). Generally speaking, I think this is a fine approach, but it may be something to consider for potential side-effects.

Reply:
Yeah, I generally lean towards enforcing that functions only be called with certain types, so in this case I'd consider calling this function with anything other than one of the 2 valid strings or `undefined` to be an error that should be caught by the linter/compiler.

I initially tried to use a type union of `('assertive'|'polite')` as the `ariaLive` JSDoc type, but despite something similar appearing as [an example in our docs](https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#custom-types), the auto-generated docs weren't handling it properly and was just outputting `(|)` I'm going to make another PR for that change later and see if anyone can figure out how to fix the issue.

Reply:
> I initially tried to use a type union of `('assertive'|'polite')` as the `ariaLive` JSDoc type, but despite something similar appearing as [an example in our docs](https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#custom-types), the auto-generated docs weren't handling it properly and was just outputting `(|)` I'm going to make another PR for that change later and see if anyone can figure out how to fix the issue.

Yeah, this is related to the issue described at #18045, and tangentially #19952 (https://github.com/WordPress/gutenberg/issues/19952#issuecomment-602376279).